corehq/apps/app_manager/views/releases.py

```diff
@@ -649,3 +663,40 @@ def toggle_build_profile(request, domain, build_id, build_profile_id):
                 build.build_profiles[build_profile_id].name
             ))
     return HttpResponseRedirect(reverse('download_index', args=[domain, build_id]))
+
+
+@require_deploy_apps
+def paginate_release_logs(request, domain, app_id):
+    limit = request.GET.get('limit')
+    page = int(request.GET.get('page', 1))
+    page = max(page, 1)
+    try:
+        limit = int(limit)
+    except (TypeError, ValueError):
+        limit = 5
+
+    app_release_logs = ApplicationReleaseLog.objects.filter(app_id=app_id).order_by('-timestamp')
+    paginator = Paginator(object_list=app_release_logs, per_page=limit)
+    current_page = paginator.get_page(page)
+
+    timezone = get_timezone_for_user(request.couch_user, domain)
+    transformed_logs = list(populate_data_app_release_logs(log, timezone) for log in current_page)
+
+    return JsonResponse({
+        'app_release_logs': transformed_logs,
+        'pagination': {
+            'total': paginator.count,
+            'num_pages': paginator.num_pages,
+            'current_page': page,
+            'more': page * limit < 10,
+        }
+    })
+
+
+def populate_data_app_release_logs(log, timezone):
+    timestamp = ServerTime(log.timestamp).user_time(timezone)
+    timestamp_date = timestamp.ui_string(USER_DATE_FORMAT)
+    timestamp_time = timestamp.ui_string(USER_TIME_FORMAT)
+    log.timestamp = "{} {}".format(timestamp_date, timestamp_time)
```

Comment:
can this be rendered together in `ui_string` in a single call? Not sure why we need to do date and time separately.

Reply:
We already spoke about this offline,so just noting to remember for code review. 
We should not modify the original object.